xen-posix/src/mini_libc.c

```diff
@@ -321,4 +323,10 @@ unsupported_function_crash(read);
 unsupported_function_crash(gmtime);
 unsupported_function_crash(strtod);
 unsupported_function_crash(rename);
-unsupported_function_crash(strerror);
+
+unsupported_function_log(size_t, fwrite, 1);
+unsupported_function_log(int, fputc, 1);
+unsupported_function_log(int, fputs, 1);
```

Comment:
( Ideally, we wouldn't have any crashing or logging stubs and the calling code would be updated to support Mini-OS properly. So I'd prefer to avoid adding more stubs if we can avoid it. That way, we'll get link-time errors rather than runtime failures. However, sometimes implementing them is the easiest way out, but I'd like to understand why we need them if so. )